pyocd/debug/rtt.py

```diff
@@ -436,20 +433,20 @@ def _find_control_block(self) -> Optional[int]:
         offset: int = 0
 
         while search_size:
-            read_size = min(search_size, 32)
-            data = self.target.read_memory_block32(addr, read_size)
+            read_size = min(search_size, 8)
```

Comment:
Reducing the read size will slow the search down noticeably. 
It's probably best to leave this at 32 for now.

(Really, the read size should be at least 512 or 1024 bytes for best performance, especially with HS USB probes. However, with large sizes like that you need to take into account the greater likelihood of running off the end of the memory region when not using the default RAM region.)